src/Http/Controllers/Support/ResolveController.php

```diff
@@ -122,4 +123,28 @@ public function resolveIdsToNames(Request $request)
 
         return response()->json($response);
     }
+
+    public function resolveMainCharacter(Request $request)
```

Comment:
Can we move this into a common class as it's duplicated here https://github.com/eveseat/web/pull/194/files#diff-efb6a0fe2c9e632d6125118d1f284731R127 ?

Maybe https://github.com/eveseat/web/blob/master/src/Http/Controllers/Support/ResolveController.php ?